### ISSUE TITLE
ng xi18n fails with Node 12.0

### ISSUE THREAD

**Issue by istiti** [angular/angular-cli#14351]
# 🐞 Bug report

```
- [x] xi18n
```

### Is this a regression?

Yes it was working in node <12

### Description
With Node 12.0.0 export i18n isn't working, due to this issue https://github.com/nodejs/node/issues/27379 on nodejs side 

## 🔬 Minimal Reproduction
Run `ng xi18n --i18n-format=xmb --output-path locale`

## 🔥 Exception or Error

```
> ng xi18n --i18n-format=xmb --output-path locale



#
# Fatal error in , line 0
# Check failed: U_SUCCESS(status).
#
#
#
#FailureMessage Object: 0000001B026FCF40npm ERR! code ELIFECYCLE
```

## 🌍 Your Environment
Angular CLI: 8.0.0-rc.2
Node: 12.0.0
OS: win32 x64
Angular: 8.0.0-rc.2
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.800.0-rc.2
@angular-devkit/build-angular     0.800.0-rc.2
@angular-devkit/build-optimizer   0.800.0-rc.2
@angular-devkit/build-webpack     0.800.0-rc.2
@angular-devkit/core              8.0.0-rc.2
@angular-devkit/schematics        8.0.0-rc.2
@angular/cdk                      8.0.0-rc.0
@angular/http                     8.0.0-beta.10
@angular/material                 8.0.0-rc.0
@angular/pwa                      0.800.0-rc.2
@ngtools/webpack                  8.0.0-rc.2
@schematics/angular               8.0.0-rc.2
@schematics/update                0.800.0-rc.2
rxjs                              6.5.1
typescript                        3.4.5
webpack                           4.30.0

**Anything else relevant?**
## It is fixed in NodeJS v.12.1.0 in [this PR](https://github.com/nodejs/node/pull/27415)

cc @filipesilva

**Comment by clydin** [comment]
This should now be corrected in Node.js 12.1.0 or greater ([changelog](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md#2019-04-29-version-1210-current-targos).

**Comment by filipesilva** [comment]
Closing as fixed by Node 12.1
